site/_shortcodes/Aside.js

```diff
@@ -0,0 +1,59 @@
+const fs = require('fs');
+const {i18n} = require('../_filters/i18n');
+
+/**
+ * Load SVG icons to be injected into the page.
+ * @param {string} name The filename (without extension) of the svg icon to
+ * load.
+ * @return {string} The SVG file contents.
+ */
+const loadIcon = name =>
+  fs.readFileSync(`site/_includes/icons/${name}.svg`, 'utf-8');
+
```

Comment:
Maybe this could be a helper function since other components will likely need icons too.

Reply:
Good idea. Because we're a little tight on time today I think i'm going to go ahead and land this and we can look at extracing the util out in a separate PR